rooms/Room1/Room1.yy

```diff
@@ -27,10 +27,10 @@
   "resourceType":"GMRoom",
   "resourceVersion":"2.0",
   "roomSettings":{
-    "Height":768,
+    "Height":720,
     "inheritRoomSettings":false,
     "persistent":false,
-    "Width":1366,
+    "Width":1280,
```

Comment:
Yep this pr worked. Though since it's a test I won't be approving it for merging.